### PR TITLE
Github actions for make

### PR DESCRIPTION
**Pull request by frederikja163**
Sets up github actions, so it runs our tests on all pull requests.

### CODE REVIEW

**Comment by frederikja163** [comment]
Would like a review here whenever you are available @AleksanderRDL @Aavild

**Review by Aavild — APPROVED** [review]

**Review by Aavild — APPROVED** [review]

**Review by Emilorzz — COMMENTED** [review]
good job

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+# CO2
+
+This is a opensource project developed by a group of university students at AAU. The goal is to calculate when home appliances have to start in order to use as little CO2 as possible.
+
+## Building
+Make sure you have a recent version of gcc and make insalled.
+The project is written using gcc and ansi C, CuTest is used for unit testing.
+`make`/`make build-run` will build and run the program and all tests automatically. But there are other targets you can use too.
+
+* `make build-shared` compiles the files in the shared folder.
```
Comment (frederikja163) [inline README.md:10]: This is interesting

**Review by Aavild — APPROVED** [review]

**Review by frederikja163 — COMMENTED** [review]
This is a very good PR

**Review by Emilorzz — APPROVED** [review]
very good

**Review by AleksanderRDL — APPROVED** [review]
Looks good, men hvorfor kommer der mælk ud af min penis?

**Review by Emilorzz — APPROVED** [review]
Jeg er til mænd

File: .github/workflows/c-cpp.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: C/C++ CI
+
+on:
+  pull_request:
+    branches: [ main ]
+
```
Comment (Aavild) [inline .github/workflows/c-cpp.yml:6]: Den her linje er ret fræk

File: README.md (hunk 1 above)
Comment (AleksanderRDL) [reply README.md:10]: Yes (づ｡◕‿‿◕｡)づ

**Review by Vaalmigi — APPROVED** [review]
Very Nice, Prove me wrong sir!

**Review by Aavild — APPROVED** [review]
Frankerne gjorde Vikingerne bange og jeg er til mænd

**Review by Ajay2407 — APPROVED** [review]
I want daddy to make me scream as he plays with my wet ass

File: README.md (hunk 1 above)
Comment (Emilorzz) [reply README.md:10]: hey this is library

Comment (Vaalmigi) [reply README.md:10]: Your mum is also very interesting

**Pull request merged**